src/App.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
 import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Calculator from './components/Calculator';
+import NavBar from './components/NavBar';
+import Home from './pages/Home';
+import Quote from './pages/Quote';
 
 const App = () => (
-  <div>
-    <Calculator />
-  </div>
+  <Router>
+    <NavBar />
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/Calculator" element={<Calculator />} />
```

Comment:
- [ ] I kindly suggest you style this page according to the wireframe as well. 

## Wireframe 
![png-transparent-nigerian-naira-currency-symbol-symbol-miscellaneous-angle-text](https://user-images.githubusercontent.com/103386183/216787915-8b025ce6-4d4c-45cb-bd30-1e6de8a27c1f.png)